src/main/java/org/candlepin/subscriptions/tally/InventoryAccountUsageCollector.java

```diff
@@ -69,7 +69,8 @@ public InventoryAccountUsageCollector(FactNormalizer factNormalizer,
         this.culledOffsetDays = props.getCullingOffsetDays();
     }
 
-    @SuppressWarnings("squid:S3776")
+    //squid:S3776 is the cognitive complexity violation in sonar
+    @SuppressWarnings({ "squid:S3776", "methodlength" })
```

Comment:
Suppressed MethodLength violation.  Limit is 100, and with my change just for readability it bumped it up to 204.  Between the method length and the cognitive complexity violation, is it almost time to refactor this method?